erpnext/patches/v14_0/update_opportunity_currency_fields.py

```diff
@@ -6,8 +6,8 @@
 
 
 def execute():
-	frappe.reload_doc('crm', 'doctype', 'opportunity')
-	frappe.reload_doc('crm', 'doctype', 'opportunity_item')
+	frappe.reload_doc('crm', 'doctype', 'opportunity', force=True)
+	frappe.reload_doc('crm', 'doctype', 'opportunity_item', force=True)
```

Comment:
shouldn't be required!

We use hash-based migration now, so if the file is changed it should reload.